src/main/java/com/aetherteam/aetherii/item/equipment/armor/abilities/GravititeArmor.java

```diff
@@ -30,7 +30,14 @@ static void playerUpdate(PlayerTickEvent.Post event) {
         if (player.onGround() && attachment.isGravititeJumpUsed()) {
             attachment.setGravititeJumpUsed(false);
         }
-        if (!player.onGround() && accessor.aether$isJumping() && accessor.aether$getNoJumpDelay() == 0 && EquipmentUtil.hasArmorAbility(player, AetherIIArmorMaterials.GRAVITITE) && !attachment.isGravititeJumpUsed()) {
+        if((player.isInWater() || player.isInFluidType())){
```

Comment:
To match Aether's code styling, would you be able to put a space in between places like before and after parentheses? E.g. `if((player` -> `if ((player`. In that example you could also change the double parentheses to single for cleanliness.